stew/byteutils.nim

```diff
@@ -246,12 +246,11 @@ func toBytes*(s: string): seq[byte] =
   ## nim essentially are byte sequences without any particular encoding, this
   ## simply copies the bytes without a null terminator
   when nimvm:
-    var r = newSeq[byte](s.len)
```

Comment:
Have you confirmed this also happens `when nimvm`? They're entirely different execution environments.

Reply:
Do you want me to keep the `var r =` part? I changed it to `result` to be consistent with the else branch.